tuxemon/states/monster/__init__.py

```diff
@@ -25,20 +25,30 @@
 #
 #
 # states.MonsterMenuState Handles creating monster menu objects
+# states.JournalMenuState Menu that shows list of all monsters
 #
 from __future__ import annotations
+import logging
 import pygame
 
 from pygame.rect import Rect
 from tuxemon import prepare, graphics
+from tuxemon.locale import T
```

Comment:
Try to obey https://peps.python.org/pep-0008/#:~:text=Imports%20are%20always%20put%20at,Related%20third%20party%20imports.